src/protocol/contract.rs

```diff
@@ -1292,8 +1285,8 @@ mod test {
                 next_hashlock_pubkey: pub_1,
                 next_multisig_pubkey: pub_2,
             }],
-            next_locktime: u16::default(),
-            next_fee_rate: u64::default(),
+            next_locktime: locktime::relative::LockTime,
+            next_fee_rate: FeeRate,
```

Comment:
Here , You have to pass the `value`  of a particular type  not the `type` itself.